TwitchLib.Api.Helix.Models/Webhooks/Subscription.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace TwitchLib.Api.Helix.Models.Webhooks
+{
+    public class Subscription
```

Comment:
Might be best to add summaries to publicly available models, easier to add them now than to add them later.